ci/install.sh

```diff
@@ -10,6 +10,9 @@ install_c_toolchain() {
             sudo apt-get install -y --no-install-recommends \
                  gcc-aarch64-linux-gnu libc6-arm64-cross libc6-dev-arm64-cross
             ;;
+	*-musl)
+	    sudo apt-get install musl-tools
```

Comment:
It seems like install.sh is called before musl_packages, see log:
https://travis-ci.org/japaric/rust-everywhere/jobs/163442305